mitmproxy/addonmanager.py

```diff
@@ -77,11 +79,11 @@ def add_option(
         if name in self.master.options:
             existing = self.master.options._options[name]
             same_signature = (
-                existing.name == name and
-                existing.typespec == typespec and
-                existing.default == default and
-                existing.help == help and
-                existing.choices == choices
+                    existing.name == name and
+                    existing.typespec == typespec and
+                    existing.default == default and
+                    existing.help == help and
+                    existing.choices == choices
```

Comment:
Does the diff just look weird, or is this now 8 spaces indented? (same question for above)

Reply:
This is just auto-formatted with PyCharm... Does your vscode/black/$IDE undo that? If yes, then let's undo it. If no, let's keep it as the pragmatic choice.